model/notification.py

```diff
@@ -6,17 +6,18 @@
 
 
 class Notification(db.Model):  # type: ignore
-    def __init__(self):
+    def __init__(self, user):
         self.id = str(uuid.uuid4())
         self.created_at = datetime.datetime.now(timezone.utc).timestamp()
+        self.user = user
 
     __tablename__ = "notification"
     id = db.Column(db.CHAR(36), primary_key=True)
-    title = db.Column(db.String, nullable=True)
-    message = db.Column(db.String, nullable=True)
-    type = db.Column(db.String, nullable=True)
-    target = db.Column(db.String, nullable=True)
-    read = db.Column(db.BOOLEAN, nullable=True)
+    title = db.Column(db.String, nullable=False)
+    message = db.Column(db.String, nullable=False)
+    type = db.Column(db.String, nullable=False)
+    target = db.Column(db.String, nullable=False)
```

Comment:
I could see this being nullable. ( if there is just a minor warning/announcement perhaps)

Reply:
this is in reference to the `target` field